tensorboard/components/tf_dashboard_common/tf-collapsable-pane.html

```diff
@@ -20,13 +20,11 @@
 
 <dom-module id="tf-collapsable-pane">
   <template>
-    <button
-      class="heading"
-      on-tap="togglePane"
-      open-button$="[[opened]]"
-    >
-    <span class="name">[[name]]</span>
-    <span class="count">
+    <button class="heading"
+            on-tap="togglePane"
+            open-button$="[[opened]]">
```

Comment:
I find it difficult to read Facebook style HTML and it's incompatible with Google style.

![image](https://user-images.githubusercontent.com/49262/27411985-5da88188-56a6-11e7-90f7-2a61d04acf6c.png)